src/async_fuse/memfs/cache/mod.rs

```diff
@@ -972,7 +972,13 @@ mod test {
         let global = GlobalCache::new();
         let file_name = "test_file";
         let content = AlignedBytes::new_from_slice(&[b'a'], 1);
-        smol::block_on(global.write_or_update(file_name.as_bytes(), 0, 1, &content, true));
+        futures::executor::block_on(global.write_or_update(
```

Comment:
change test function to async instead of using block_on() here

Reply:
done